.github/workflows/cd_android.yml

```diff
@@ -40,6 +40,12 @@ jobs:
           versionCode=$GITHUB_RUN_NUMBER; echo "versionName: $versionName; versionCode: $versionCode"
           echo "export VERSION_NAME=$versionName; export VERSION_CODE=$versionCode" > ~/.env_version
 
+      - name: Create internal environment
+        env:
+          ENV: ${{secrets.ENV}}
+        run: |
+          echo $ENV > .env
```

Comment:
Do you need to delete it after CI run is complete?

Reply:
no, all the images are discarded automatically when pipeline is done